Tracking/DisasterTrackerApp.Models/Disaster/EventDto.cs

```diff
@@ -1,13 +1,30 @@
+using System.Text.Json.Serialization;
+
 namespace DisasterTrackerApp.Models.Disaster;
 
 public class EventDto
 {
+    [JsonPropertyName("id")]
     public string Id { get; set; }
+
+    [JsonPropertyName("title")]
     public string Title { get; set; }
+
+    [JsonPropertyName("description")]
     public string Description { get; set; }
+
+    [JsonPropertyName("link")]
     public string Link { get; set; }
-    public DateTime? Closed { get; set; }
+
+    [JsonPropertyName("closed")]
+    public DateTime Closed { get; set; }
```

Comment:
```suggestion
    public DateTime? Closed { get; set; }
```

"Closed": null 
when disaster event is active